rts/motoko-rts/src/gc/mark_compact/bitmap.rs

```diff
@@ -4,37 +4,45 @@ use crate::types::{size_of, Blob, Bytes, Obj};
 
 /// Current bitmap
 static mut BITMAP_PTR: *mut u8 = core::ptr::null_mut();
+static mut BITMAP_COMPENSATION: usize = 0;
+static mut BITMAP_SIZE: u32 = 0;
```

Comment:
You will be the maintainer so feel free to ignore, but documenting these variables would be good. For example, that the `BITMAP_SIZE` is only used for sanity checking.

I find it a bit difficult to explain what `BITMAP_COMPENSATION` is, maybe a visualization (in ASCII) would help.

I think it's also worth documenting why you choose to implement this with these set of variables. Here's an alternative implementation:

- BITMAP_PTR: pointer to the blob payload. Used in `bitmap_iter`. Saves us two cycles when skipping a bitmap word in `BitmapIter::next`.
- BLOB_PTR: pointer to the blob header. Used in sanity checks to get the blob size.
- BITMAP_PTR_SHIFTED: (not sure about the name) the base address used in `set_bit` and `get_bit`.

Reply:
This PR is still draft and evolving. I used the current schema, because it was the first I got working. I am aware of the pointer pair representation, and will probably switch. Nice ASCII graphics coming.

Reply:
Ah, sorry, didn't realize this is draft.

Reply:
Okay, we have renamed variables, PTAL.